src/ansys/dpf/core/_version.py

```diff
@@ -23,7 +25,7 @@ class ServerToAnsysVersion:
         "7.1": "2024R1",
         "8.0": "2024R2",
         "8.1": "2024R2",
-        "8.2": "2024R2",
```

Comment:
Why do we remove this line? @guptadivya1501

Reply:
@BClappe , Oops I dont know how it is removed. I will add again